src/randoop/operation/OperationParser.java

```diff
@@ -8,24 +8,28 @@
 
   /**
    * Parses a string representing a StatementKind. The string is expected to be
-   * of the form:
+   * of the form:<br>
```

Comment:
In some places in this javadoc comment you added <br> tags and in other places <p> tags.  Is there a reason for the difference?